### ISSUE TITLE
aws inf1.2xlarge with inferentia only use one neuroncore of four

### ISSUE THREAD

**Issue by yingmuting** [deepjavalibrary/djl#1201]
## Description
在inf1.2xlarge上deploy demo benchmark：https://docs.djl.ai/docs/demos/aws/inferentia/index.html
<img width="842" alt="图片" src="https://user-images.githubusercontent.com/20216265/131655376-14cb4ae7-4f1c-4341-bf80-7c0072d6e7db.png">

10000次推理吞吐量284.39/s，inf1.2xlarge有一个inferentia芯片，4个neuroncore，通过neuron-ls可以查看
<img width="601" alt="图片" src="https://user-images.githubusercontent.com/20216265/131654852-93fdb225-95e2-4675-a267-a26038e21d4b.png">
，推理时无法使用4个neuroncore，导致推理吞吐量上不去，
<img width="659" alt="图片" src="https://user-images.githubusercontent.com/20216265/131655173-9108de20-3e0b-4e6e-bed2-137628ba84a2.png">

相比较下inf1.2xlarge跑同样的python推理，吞吐量可以达到2000/s以上

### Expected Behavior
吞吐量接近python

**Comment by frankfliu** [comment]
@yingmuting 
You can set environment vairable: `NEURONCORE_GROUP_SIZES` to control how many neuroncore to be used for your model. See: https://awsdocs-neuron.readthedocs-hosted.com/en/latest/neuron-guide/neuron-frameworks/tensorflow-neuron/tutorials/tutorial-tensorflow-NeuronCore-Group.html

**Comment by frankfliu** [comment]
NEURONCORE_GROUP_SIZES is generic for both pytorch and tensorflow

**Comment by yingmuting** [comment]
when compile model to neuron ,the command is "model_neuron = torch.neuron.trace(model, example_inputs=[image], verbose=1, compiler_args = ['--neuroncore-pipeline-cores', '4'])"，load the model to djl springboot demo，it can run with 4 neuroncore on inf1，but it not work on eks with neuron-rtd container and neuron-device-plugin，even set the NEURONCORE_GROUP_SIZES env on container

**Comment by yingmuting** [comment]
Add error message:
Caused by: ai.djl.engine.EngineException: NeuronDevice::process_nrtd_response: Context=NRTD_CTX_LOAD NRTD response code 'NERR_FAIL' (1) This is a general catch-all failure.  Please reach out to AWS support. Raw runtime error message '[TDRV:model_dma_engine_and_queue_alloc] no more rings
[TDRV:kbl_model_add] create_engine_refill_rings_tonga() error
[NMGR:dlr_kelf_stage] Failed to load subgraph
[NMGR:stage_kelf_models] Failed to stage graph: kelf-a.json to NeuronCore
[NMGR:kmgr_load_nn] Failed to load NN: 1.4.1.0+737cbb69a-/tmp/tmpri956cec, err: 1
'
        at ai.djl.pytorch.jni.PyTorchLibrary.moduleLoad(Native Method) ~[pytorch-engine-0.12.0.jar!/:?]
        at ai.djl.pytorch.jni.JniUtils.loadModule(JniUtils.java:1359) ~[pytorch-engine-0.12.0.jar!/:?]
        at ai.djl.pytorch.engine.PtModel.load(PtModel.java:85) ~[pytorch-engine-0.12.0.jar!/:?]
        at ai.djl.repository.zoo.BaseModelLoader.loadModel(BaseModelLoader.java:156) ~[api-0.12.0.jar!/:?]
        at ai.djl.repository.zoo.Criteria.loadModel(Criteria.java:174) ~[api-0.12.0.jar!/:?]

**Comment by jeffhataws** [comment]
Hello yingmuting, I just want to confirm that this model compiles with --neuroncore-pipeline-cores=4 and run on inf1, but fails to work on eks with the above error message? Could you share more details with us to help debug? You can contact us directly aws-neuron-support@amazon.com.

**Comment by yingmuting** [comment]
> Hello yingmuting, I just want to confirm that this model compiles with --neuroncore-pipeline-cores=4 and run on inf1, but fails to work on eks with the above error message? Could you share more details with us to help debug? You can contact us directly [aws-neuron-support@amazon.com](mailto:aws-neuron-support@amazon.com).

ok，i will send email，include details，thx。

**Comment by aws-renfu** [comment]
Hi yingmuting,

We did a quick test, but could not reproduce. This is what we did:

1. compiled the model using the script from the overview
2. extracted the neff
3. inspected the neff - it has 4 subgraphs (as expected) every SG uses at most 2 input, 2 output and 4 data DMA rings which is supported configuration.
4. tried loading the neff - it loads successfully.

The test was run on DLAMI Deep Learning AMI (Ubuntu 18.04) Version 49.0
aws-neuron-runtime                     1.5.0.0
(aws_neuron_pytorch_p36) ubuntu@ip-172-31-0-229:~$ neuron-cc --version
Neuron Compiler version 1.6.13.0+9f61b2f75
HWM version 1.6.0.0-0
NEFF version Dynamic
TVM version 1.6.2.0+0
NumPy version 1.19.5

We would suggest rebuilding the container using the most recent version of Neuron.
